### DISCUSSION TITLE
Color Blindness Accessibility

### DISCUSSION THREAD

**Discussion by dr-dirt** [OpenRCT2/OpenRCT2#19909] [Feature requests]
I've noticed that there aren't accessibility options for color blindness.  The expanded colors are arranged in a way that makes it quite difficult for red/green color deficiency to distinguish between some of the colors on the palette, especially in the area of brown/peach/olive.  It would also be nice to have the interface colors with alternate options, especially the scenery window can provide difficulty with its green background.  In actuality, most of the windows are green, brown, peach-ish in color which are all difficult colors for users with red/green color deficiency.  It would affect a significant number of users: for males, it's around 5-8% that have this deficiency, and there are other deficiencies that also could be improved for accessibility with this feature.

**Comment by IntelOrca** [comment]
You can already change the interface theme. For example the RCT1 theme uses more grey colours.
The colour picker also has tooltips on each colour block that states the name.

I am colour blind myself and the problem is usually when it is necessary to distinguish between two different colours, such as a match-3 type game. I have not personally had an issue with the window colours, but as I mentioned - you can already change them and create custom themes. I will try using the new colour picker to see if I have any issues with that.

**Reply by dr-dirt** [reply]
I just discovered the themes could be changed shortly after posting this.  Perhaps some premade themes for this purpose would be quite nice to have.  For example, just changing the scenery window to the periwinkle makes it quite a bit easier to see the color picker already.  It might be a worthwhile consideration to offer these by default, so someone isn't required to fully customize everything.

**Reply by tupaschoal** [reply]
Would you be willing to develop a high contrast theme? That's something I think we can incorporate as a built-in theme for accessibility

**Reply by dr-dirt** [reply]
Yes, I can definitely work on something like this.
